src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -697,4 +697,13 @@
   <data name="PlainText" xml:space="preserve">
     <value>Plain Text</value>
   </data>
+  <data name="CloseOnExitInfoBar.Message" xml:space="preserve">
+    <value>Termination behavior can be configured in advanced profile settings</value>
+  </data>

```

Comment:
Idea: I think it'd be cool if (later) we made "advanced profile settings" a hyperlink that opened the settings UI for this profile and navigated to the relevant page.

The only reason I bring that up now is that if we want to pursue this, we should maybe break up this resource into two:
- "Termination behavior can be configured in {}"
- "advanced profile settings"
And in TerminalPage.cpp, we replace `{}` with "advanced profile settings" and (later) hyperlink-ify it.

Thoughts?

Reply:
I'm gonna say this is something that can be done in a follow up

Reply:
Will leave this as a follow up as it really depends on what UX we implement (i.e., probably we will need to put the hyperlink as a separate element)